pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -478,30 +487,352 @@ func (p *mockProvider) OutboundTransport() transport.OutboundTransport {
 	return didcomm.NewMockOutboundTransport(successResponse)
 }
 
-//TODO: Use in memory store rather than level DB " https://github.com/hyperledger/aries-framework-go/issues/202
-func store(t testing.TB) (store storage.Store, cleanup func()) {
-	path, cleanup := tempDir(t)
-	prov, err := leveldb.NewProvider(path)
-	require.NoError(t, err)
+func store(t testing.TB) (storage.Store, func()) {
+	prov := mockstorage.NewMockStoreProvider()
 	dbstore, err := prov.GetStoreHandle()
 	require.NoError(t, err)
-	return dbstore, cleanup
+	return dbstore, func() {
+		err := prov.Close()
+		require.NoError(t, err)
+	}
+}
+
+func randomString() string {
+	u := uuid.New()
```

Comment:
same here

Reply:
Discussion is tracked [here](https://github.com/hyperledger/aries-framework-go/pull/240#discussion_r324806191).